README.md

```diff
@@ -16,13 +16,14 @@
 
 ## Axolotl supports
 
-|         | fp16/fp32 | fp16/fp32 w/ lora | qlora | 4bit-quant | 4bit-quant w/flash attention | flash attention | xformers attention |
-|---------|:----------|:------------------|------|------------|------------------------------|-----------------|--------------------|
-| llama   | ✅         | ✅                 | ✅  | ✅          | ✅                            | ✅               | ✅                  |
-| Pythia  | ✅         | ✅                 | ❓  | ❌          | ❌                            | ❌               | ❓                  |
-| cerebras | ✅         | ✅                 | ❓  | ❌          | ❌                            | ❌               | ❓                  |
-| mpt     | ✅         | ❌                 | ❓  | ❌          | ❌                            | ❌               | ❓                  |
-| falcon  | ✅         | ✅                 | ✅  | ❌          | ❌                            | ❌               | ❓                  |
+|          | fp16/fp32 | fp16/fp32 w/ lora | qlora | 4bit-quant | 4bit-quant w/flash attention | flash attention | xformers attention |
+|----------|:----------|:------------------|------|------------|------------------------------|----------------|-----------------|
+| llama    | ✅         | ✅                 | ✅  | ✅          | ✅                            | ✅              | ✅               |
+| Pythia   | ✅         | ✅                 | ❓  | ❌          | ❌                            | ❌              | ❓               |
+| cerebras | ✅         | ✅                 | ❓  | ❌          | ❌                            | ❌              | ✅                |
+| mpt      | ✅         | ❌                 | ❓  | ❌          | ❌                            | ❌              | ❓               |
+| falcon   | ✅         | ✅                 | ✅  | ❌          | ❌                            | ❌              | ✅                |
+| gpt-j    | ✅         | ✅                 | ✅  | ❌          | ❌                            | ❓               | ✅                |
```

Comment:
should we add gpt Neo x? I did see that you deleted it though.

Reply:
yeah, gpt-neox needs some more time and love (technically, pythia is gpt-neox, but the gpt-neox-20b gives me headaches all day)